File: curso-udemy/curso-algoritmo-completo/Módulo 9 - C++/Exercícios/EXCIDADE.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    
    int primeiraIdade, segundaIdade;
    string primeiroNome, segundoNome;
    double mediaIdade;

    cout << "Digite o primeiro nome:";
    getline(cin, primeiroNome);
    cout << "Digite a idade desta pessoa:";
    cin >> primeiraIdade;

    cout << "Digite o segundo nome:";
    cin.ignore(INT_MAX, '\n');
    getline(cin, segundoNome);
    cout << "Digite a segunda idade:";
    cin >> segundaIdade;

    mediaIdade = (double) (primeiraIdade + segundaIdade) / 2.0;
    
    cout << fixed << setprecision(2);
    cout << "A média entre " << primeiroNome << " e " << segundoNome << " é de " << mediaIdade << " anos.";

    return 0;
}

File: programming-logic/algorithm/knowing-java/Exercícios/exercicio_java/src/exercicio_java/EXCIDADE.java
package exercicio_java;
import java.util.Locale;
import java.util.Scanner;

public class EXCIDADE {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int primeiraIdade, segundaIdade;
		double mediaIdade;
		String primeiroNome, segundoNome;
		
		System.out.println("Qual o nome da 1ª pessoa?");
		System.out.print("Nome: ");
		primeiroNome = sc.nextLine();
		System.out.print("Idade: ");
		primeiraIdade = sc.nextInt();
		
		sc.nextLine();
		
		System.out.println("Qual o nome da 2ª pessoa?");
		System.out.print("Nome: ");
		segundoNome = sc.nextLine();
		System.out.print("Idade: ");
		segundaIdade = sc.nextInt();	
		
		mediaIdade = (double) (primeiraIdade + segundaIdade) / 2;
		
		System.out.println("A idade média entre " + primeiroNome + " e " + segundoNome + " é de " + String.format("%.1f", mediaIdade) + " anos.");
		
		sc.close();
		

	}

}
